docs/resources/dli_flinkjob_jar.md

```diff
@@ -0,0 +1,153 @@
+  You can add the following content to the application to access the corresponding dependency file: In the command,
+  fileName indicates the name of the file to be accessed, and ClassName indicates the name of the class that needs to
+  access the file: ClassName.class.getClassLoader().getResource("userData/fileName").
+
+* `feature` - (Optional, String) Specifies job feature. Type of the Flink image used by a job.
+  + **basic**: indicates that the basic Flink image provided by DLI is used.
+  + **custom**: indicates that the user-defined Flink image is used.
+
+  The default value is **basic**.
+  
+* `flink_version` - (Optional, String) Specifies flink version. This parameter is valid only when feature is set
+ to basic. You can use this parameter with the feature parameter to specify the version of the DLI basic Flink image
+ used for job running. The options are as follows: `1.10` and `1.7`.
+
+* `image` - (Optional, String) Specifies custom image. The format is Organization name/Image name:Image version.
+  This parameter is valid only when feature is set to `custom`. You can use this parameter with the feature parameter
+  to specify a user-defined Flink image for job running. For details about how to use custom images, see the
+  Data Lake Insight User Guide <https://support.huaweicloud.com/en-us/usermanual-dli/dli_01_0494.html>.
+  
+* `cu_number` - (Optional, Int) Specifies number of CUs selected for a job. The default value is `2`.
+
+* `parallel_number` - (Optional, Int) Specifies number of parallel for a job. The default value is `1`.
+
+* `manager_cu_number` - (Optional, Int) Specifies number of CUs in the JobManager selected for a job.
+ The default value is `1`.
+  
+* `tm_cus` - (Optional, Int) Specifies number of CUs for each TaskManager. The default value is `1`.
+  
+* `tm_slot_num` - (Optional, Int) Specifies number of slots in each TaskManager.
+ The default value is `(parallel_number * tm_cus) / (cu_number - manager_cu_number)`.
+
+* `smn_topic` - (Optional, String) Specifies SMN topic. If a job fails, the system will send a message to users
+  subscribed to the SMN topic.
+
+* `log_enabled` - (Optional, Bool) Specifies whether to enable the function of uploading job logs to users' OBS buckets.
+ The default value is `false`.
+
+* `obs_bucket` - (Optional, String) Specifies OBS path. OBS path where users are authorized to save the log.
+  This parameter is valid only when `log_enabled` is set to `true`.
+
+* `restart_when_exception` - (Optional, Bool) Specifies whether to enable the function of restart upon exceptions.
+ The default value is `false`.
+  
+* `resume_checkpoint` - (Optional, Bool) Specifies whether the abnormal restart is recovered from the checkpoint.
+  
+* `resume_max_num` - (Optional, Int) Specifies maximum number of retry times upon exceptions. The unit is
+ `times/hour`. Value range: `-1` or greater than `0`. The default value is `-1`, indicating that the number of times is
+ unlimited.
```

Comment:
Suggest to maintain a uniform naming style for this parameters with `_num` or `_number`, e.g.:
`cu_num`, `parallel_num`, `manager_cu_num`, `tm_cu_num`, `tm_slot_num`, `resume_max_num`